services/perfLogger/perfLogger.ts

```diff
@@ -0,0 +1,155 @@
+import { Singleton } from "typescript-ioc";
+import * as _ from "lodash";
+import { Inject } from "typescript-ioc";
+import { DataBaseSDK } from "../../sdks/DataBaseSDK";
+import SettingSDK from '../../sdks/SettingSDK';
+import { logger } from "@project-sunbird/logger";
+import { ClassLogger } from "@project-sunbird/logger/decorator";
+import { TelemetryInstance } from './../telemetry/telemetryInstance';
+import { IPerfLog } from './IPerfLog';
+import { timer } from 'rxjs';
+import { EventManager } from "@project-sunbird/ext-framework-server/managers/EventManager";
+import { SystemQueue, ISystemQueue } from './../../services/queue';
+
+const INITIAL_TRIGGER = 15 * 60 * 1000; // trigger first job after 15 min  
+const DAY_IN_MILLISECONDS = 24 * 60 * 60 * 1000; // trigger jobs every 24 hours after first trigger
+const MONTH_IN_MILLISECONDS = 30 * 24 * 60 * 60 * 1000; // used in archive job to remove logs which are older than last 30 days
+const LOG_QUERY_LIMIT = 1000;
+const DB_NAME = 'perf_log';
+const LAST_PERF_LOG_PROCEEDED_ON = 'LAST_PERF_LOG_PROCEEDED_ON';
+const REQUIRED_SYSTEM_QUEUE_TASK = ["IMPORT", "DOWNLOAD"]; // should be moved to openrap-sunbirded-plugin to make it customizable
+const DEFAULT_LAST_SYNC_TIME = 1585282913052;
```

Comment:
What is this time ?

Reply:
to take as default, after first process last processed time will be stored in setting sdk